EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1230,273 +1230,273 @@ QUEST_LV_0300_20160310_001229	Even if you can't trust me you should trust Vaidot
+QUEST_LV_0300_20160310_001244	Thank you. I was able to survive because of you.{nl}Dominikas... He is more cruel than the demons.
+QUEST_LV_0300_20160310_001245	It hasn't been a long time since Dominikas visited Elgos Monastery.{nl}I thought he was just an accompanying monk.{nl}
+QUEST_LV_0300_20160310_001246	Not so long after he came here, the tree vines and the {nl}demons suddenly swept the monastery annex.{nl}The villagers fled there... But, he stopped them appropriately.{nl}
+QUEST_LV_0300_20160310_001247	When I think about it, it was all his self-fabricated scenario.{nl}The raid of the tree vines and the sudden attacks of the demons...{nl}
+QUEST_LV_0300_20160310_001248	As the villagers started to follow him, he quickly showed his instinct.{nl}He started to ask us to create an evil idol and believe in the Divine Tree.{nl}
+QUEST_LV_0300_20160310_001249	Of course, we resisted... But he locked us into that black wall.{nl}He gave us an amount of water that was only enough to keep us alive{nl}and requested us to change our religion.
+QUEST_LV_0300_20160310_001250	I felt so painful since I didn't know what to do.{nl}Ah... The abbot!{nl}
+QUEST_LV_0300_20160310_001251	The abbot must be locked inside the monastery.{nl}I was hoping that the goddesses would protect us... {nl}But we will be in trouble if we don't quickly give you the food.
+QUEST_LV_0300_20160310_001252	Elgos Abbot
+QUEST_LV_0300_20160310_001253	This energy... Yes, it's you. The Revelator.{nl}It seems the goddess finally answered my prayer.{nl}
+QUEST_LV_0300_20160310_001254	Ah, nothing. If the other monks are alright, the food will be okay...{nl}I don't think I will be able to swallow the food.
+QUEST_LV_0300_20160310_001255	He is too old... Is he going to be alright...?
+QUEST_LV_0300_20160310_001256	You are not blocked by the black wall since you are the Revelator, as expected.
+QUEST_LV_0300_20160310_001257	The evil idol that was created by Dominikas in front of Pagarba Chapel...{nl}The black wall started to cover the monastery because of that idol.{nl}
+QUEST_LV_0300_20160310_001258	If we destroy that evil idol...{nl}There might be a good chance that the whole wall may disappear.
+QUEST_LV_0300_20160310_001259	I just don't know how I can destroy it...{nl}There is nothing I can think of at the moment.{nl}
+QUEST_LV_0300_20160310_001260	But, wait...{nl}I can sense the soul of the demons in you.{nl}
+QUEST_LV_0300_20160310_001261	That sphere you have...{nl}Is a demon's soul locked inside it?{nl}
+QUEST_LV_0300_20160310_001262	That's it.{nl}Maybe we can use this to destroy the evil idol.{nl}
+QUEST_LV_0300_20160310_001263	The idea is to purify the soul of the demons and use their power.{nl}Unfortunately, we lack the right amount of souls to destroy the idol.{nl}
+QUEST_LV_0300_20160310_001264	Can you fill the demon's soul a little more?{nl}As you already may know, defeating demons will automatically absorb their souls.
```

Comment:
"Can you fill the soul sphere a little more?"  ?  It's not that demon's soul is being filled, but it's the orb is being filled with more souls.